desktop/extensions-sdk/guides/oauth2-flow.md

```diff
@@ -51,14 +52,14 @@ window.ddClient.openExternal("https://authorization-server.com/authorize?
   &state=kH_0FdAtjCfYjOkF);
 ```
 
-## Step Two: Get the authorization code and access token
+## Step two: get the authorization code and access token
```

Comment:
I think @aevesdocker wanted to keep the title uppercase.

Reply:
Changed back. Vale is highlighting the uppercases there, maybe we should update rules for titles (but let's not change this in this PR in any case, agreed)